old_sidebars.js

```diff
@@ -0,0 +1,158 @@
+module.exports = {
```

Comment:
I assume this file got added accidentally as part of git add?

Reply:
Yes..now removed